api/scpca_portal/management/commands/sync_original_files.py

```diff
@@ -13,20 +16,49 @@
 
 class Command(BaseCommand):
     help = """
-    Sync OriginalFiles table with s3 input bucket.
+    Sync OriginalFile table with s3 input bucket.
     """
 
     def add_arguments(self, parser):
-        parser.add_argument("--bucket-name", type=str, default=settings.AWS_S3_INPUT_BUCKET_NAME)
+        parser.add_argument("--bucket", type=str, default=settings.AWS_S3_INPUT_BUCKET_NAME)
+        # if all files have been wiped in passed s3 bucket, deletion in db must be manually enabled
+        # this is mainly for testing purposes
+        parser.add_argument("--allow-bucket-wipe", type=bool, default=False)
 
     def handle(self, *args, **kwargs):
         self.sync_original_files(**kwargs)
 
-    def sync_original_files(self, bucket_name: str, **kwargs):
+    def get_indented_files(self, files: List[OriginalFile]) -> str:
+        formatted_file_str = "\n".join(f"\t{str(f)}" for f in files) if files else "\tNone"
+        return formatted_file_str
+
+    def log_file_changes(self, updated_files, created_files, deleted_files, sync_timestamp) -> None:
```

Comment:
I noticed this method is missing typing. Just a small suggestion to match the others: 

```suggestion
def log_file_changes(self, updated_files: List[OriginalFile], created_files: List[OriginalFile], deleted_files: List[OriginalFile], sync_timestamp: datatime) -> None:
```